reference/tax_settings.v3.yml

```
@@ -159,6 +159,13 @@ components:
           default: 1
           type: integer
           description: ID for the tax zone a store uses when subtracting store tax. This setting applies only if a merchant enters tax-inclusive prices and subtracts store tax before tax calculation.
+        document_submission_strategy:
+          default: ON_PAYMENT_CAPTURE
+          type: string
+          description: This setting determines whether BigCommerce submits tax documents to third-party tax providers when orders are created or when payments are captured online. 
```

I wanted to remove the passive voice but I am not positive who is doing the action. Is it the shopper or customer that creates the order and the merchant captures the payment? Thoughts?
```suggestion
          description: This setting determines whether BigCommerce submits tax documents to third-party tax providers when shoppers create orders or merchants capture payments online. 
```

---

I think merchants can also create manual orders?  @theromulans if you could weigh in on this one

---

Yes that's accurate @bc-andreadao - so I agree it's a bit odd to specify "when _shoppers_ create orders"

---

Thanks, I'll keep the current passive voice

---

I just realized I forgot to address the other question (about whether merchants are the only ones who can capture payments online). I guess maybe third-party apps can also capture payments ?

---

If a third party app captures a payment but also updates the order payment status appropriately then that's all fine.

However, if a payment is captured by a third party system and BC has no awareness of it I think it's fair that these count as "offline" payments (offline to BC) - but it's a fair point this might be confusing.